testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/multinode-all/jvm.config

```diff
@@ -18,3 +18,6 @@
 # Allow loading dynamic agent used by JOL
 -XX:+EnableDynamicAgentLoading
 -XX:+ExitOnOutOfMemoryError
+# https://bugs.openjdk.org/browse/JDK-8327134
+-Djava.security.manager=allow
+--enable-preview
```

Comment:
What needs preview features?

Reply:
Letfover. Not needed. I will remove it :)